cath_alphaflow/commands/optimise_domain_boundaries.py

```diff
@@ -235,6 +266,7 @@ def calculate_domain_id_post_tailchop(
     af_domain_id: AFDomainID,
     af_chain_mmcif_dir: Path,
     cutoff_plddt_score: int,
+    gzipped_af_chains = bool,
```

Comment:
no, this needs to be:
```
gzipped_af_chains: bool
```
or if you want to give it a default value, it should be
```
gzipped_af_chains: bool = True
```

Reply:
adding the default value would match the existing functionality, but allow a different option - which is generally what you want to do

Reply:
Done